packages/website/docs/getting-started.md

```diff
@@ -58,15 +39,21 @@ If you do not wish to use a package manager, you can use standalone endpoints:
 <script src="https://unpkg.com/@algolia/autocomplete-js@alpha"></script>
```

Comment:
In our docs [we recommend using jsDelivr](https://www.algolia.com/doc/api-client/getting-started/update-the-client/javascript/?client=javascript#other-cdns), we may want to remove this unpkg mention then. Besides, users might be tempted to copy both lines.